app/src/test/java/com/nervousfish/nervousfish/modules/pairing/events/BluetoothConnectedEventTest.java

```diff
@@ -1,10 +1,22 @@
 package com.nervousfish.nervousfish.modules.pairing.events;
 
+import com.nervousfish.nervousfish.modules.pairing.AndroidBluetoothConnectedThread;
+
 import org.junit.Test;
 
+import static org.junit.Assert.assertEquals;
+import static org.mockito.Mockito.mock;
+
 public class BluetoothConnectedEventTest {
     @Test
     public void testBluetoothConnectedEvent() {
         new BluetoothConnectedEvent(null);
     }
+
+    @Test
+    public void testBluetoothConnectedEventGetThreadNull() {
```

Comment:
Not sure how this particular test (or even this whole test class for that matter) is useful 🤔 It doesn't seem to be testing anything useful...